NYPL/authoritydata_updater
# DR-2953 minor improvements

## JIRA ticket:
- [Reindex authority data to validate Solr upgrade](https://newyorkpubliclibrary.atlassian.net/browse/DR-2953)

## Things Done:
- Install gems in Dockerfile
- Fixed documentation for pullling docs from solr
- Removed the '_version_' key from docs making them post-able
- Use error handling logic in posting to solr

## How to Test:
- `docker-compose build`
- `docker-compose run webapp bash`
- Join the VPN
- Try, for example, `bundle exec ruby pull_from_solr.rb -d http://10.225.133.217:8983/solr/authoritydata -a lcgft -o data/output/lcgft.json`. It should generate the json file specified _without_ any '_version_' key included.
